src/pyodide.js

```diff
@@ -385,7 +385,7 @@ var languagePluginLoader = new Promise((resolve, reject) => {
           div.appendChild(pre);
           element = div;
         }
-        return element;
+        return element.innerHTML;
```

Comment:
i think you might want `outerHTML` if you want to return the containing div (which has the "rendered_html" element. see:
https://extremely-alpha.iodide.io/notebooks/263/

(man i love being able to share these little demos!!! yay iodide!! :-) )

Reply:
Oops.  Indeed.